chsdi/locale/en/LC_MESSAGES/chsdi.po

```diff
@@ -1544,18 +1544,288 @@ msgstr "Zählstelle"
 msgid "ch.astra.unfaelle-personenschaeden_alle"
 msgstr "Accidents with personal injury"
 
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_de"
+msgstr "Unfalltag"
```

Comment:
no translation was provided. We always ask for 5(4) languages, but officially we must accept layers with only DE/FR. A translation program runs by @danielabrandt